analyses/cell-type-wilms-tumor-14/scripts/utils/00_preprocessing_rds_functions.R

```diff
@@ -0,0 +1,75 @@
+library(dplyr)
+library(Seurat)
+library(ggpubr)
+pre_seuratobj <- function(obj, nfeatures = 500, run_harmony = TRUE, reduction = "harmony", ndims = 50,
+                          skip_logNorm = TRUE){
+  ######## Normalize, scale, feature selection
+  if (skip_logNorm){
+    obj[["RNA"]]$data <- obj[["RNA"]]$counts
```

Comment:
I don't think we want to copy the counts, but rather just verify that the `data` slot is present (and of the correct dimensions, perhaps)?
```suggestion
    stopifnot("data layer must be present if skip_logNorm selected" = 
      "data" %in% SeuratObject::Layers(obj[["RNA"]])
    )
```